app/src/main/res/layout/fragment_signature_setting.xml

```diff
@@ -45,6 +45,14 @@
                 app:layoutManager="androidx.recyclerview.widget.LinearLayoutManager"
                 tools:itemCount="3"
                 tools:listitem="@layout/item_settings_signature" />
+
+            <com.google.android.material.button.MaterialButton
+                android:id="@+id/manageSignatures"
+                style="@style/TextButtonSecondary"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/manageSignatures" />
```

Comment:
Either make the padding of the button so the actual text of the button is aligned horizontally with the list of signatures' actual text or make it so the ripple of the button is horizontally aligned with the start of the signatures' actual text but not something in between